.editorconfig

```diff
@@ -12,7 +8,6 @@ charset = utf-8
 indent_style = space
 indent_size = 4
```

Comment:
Since we are changing this file - there's something I wanted to do for quite some time, but didn't want to it until the we made more progress on 1.0: markdown lists are formatted with a lot of space because we didn't set up `indent_size = 2` for `*.md` files.

For sure we want 4 spaces for `*.{c,m,}{js,ts}{x,}` (all TS/JS extensions - react too in docs), but for all the others, I think we might want 2 - basically for `*.{json,md,yml}` (we don't have `yaml` anymore AFAIK).